presto-main/src/main/java/com/facebook/presto/util/JsonUtil.java

```diff
@@ -82,6 +102,16 @@ public static JsonGenerator createJsonGenerator(JsonFactory factory, SliceOutput
         return factory.createGenerator((OutputStream) output);
     }
 
+    public static String getTrimmedJsonForErrorMessage(String json)
+    {
+        if (json.length() <= MAX_JSON_LENGTH_IN_ERROR_MESSAGE) {
+            return json;
+        }
+        else {
+            return json.substring(0, MAX_JSON_LENGTH_IN_ERROR_MESSAGE) + "...";
```

Comment:
I'm a little bit concerned that `...` is not obvious enough to readers of error message. What about `...(truncated)` instead?